mkt/access/migrations/0003_add_contenttools_groups.py

```diff
@@ -1,8 +1,9 @@
 from django.db import migrations
+from django.db.models.loading import get_model
 
 
 def create_addon_groups(apps, schema_editor):
-    Group = apps.get_model('Access', 'Group')
```

Comment:
and do use `apps.get_model`, never use `django.db.models.loading` in a migration, you might end up with a wrong version of the model.